cast/tests/e2e/helpers/fixtures.rs

```diff
@@ -0,0 +1,75 @@
+use crate::helpers::constants::{
```

Comment:
the same as in https://github.com/software-mansion/protostar/pull/2150#discussion_r1250601906, I don't think these are e2e tests, or are they?

Reply:
these are not tests

Reply:
right, sorry wrong file